asciidoc_reader/README.rst

```diff
@@ -21,7 +21,7 @@ Settings
 ========================================  =======================================================
 Setting name (followed by default value)  What does it do?
 ========================================  =======================================================
-``ASCIIDOC_CMD = asciidoc``               Selects which utility to use for rendering. Will
+``ASCIIDOC_CMD = 'asciidoc'``               Selects which utility to use for rendering. Will
```

Comment:
Right-hand portion of this line has been mis-indented by two spaces due to the quotation marks added to its left. Would you remove two spaces in-between in order to restore the right-hand indentation to its original, aligned position?